From 842fb2283fd917101c0bfe72a9e40f945a377ce1 Mon Sep 17 00:00:00 2001
From: Johannes Schindelin <johannes.schindelin@gmx.de>
Date: Wed, 4 Dec 2019 21:45:07 +0100
Subject: Git 2.16.6

Signed-off-by: Johannes Schindelin <johannes.schindelin@gmx.de>
(cherry picked from commit eb288bc455ac67e3ceeff90daf6f25972bb586d0)
Signed-off-by: Jonathan Nieder <jrnieder@gmail.com>
---
 Documentation/RelNotes/2.16.6.txt | 8 ++++++++
 1 file changed, 8 insertions(+)
 create mode 100644 Documentation/RelNotes/2.16.6.txt

diff --git a/Documentation/RelNotes/2.16.6.txt b/Documentation/RelNotes/2.16.6.txt
new file mode 100644
index 0000000000..438306e60b
--- /dev/null
+++ b/Documentation/RelNotes/2.16.6.txt
@@ -0,0 +1,8 @@
+Git v2.16.6 Release Notes
+=========================
+
+This release merges up the fixes that appear in v2.14.6 and in
+v2.15.4 addressing the security issues CVE-2019-1348, CVE-2019-1349,
+CVE-2019-1350, CVE-2019-1351, CVE-2019-1352, CVE-2019-1353,
+CVE-2019-1354, and CVE-2019-1387; see the release notes for those
+versions for details.
-- 
2.24.0.393.g34dc348eaf

